python-searchable/docker-compose.python-searchable.yml

```diff
@@ -0,0 +1,282 @@
+        environment:
+            EXAMPLE_HOST: ${EXAMPLE_HOST:-acra-server}
+            EXAMPLE_PORT: ${EXAMPLE_PORT:-9393}
+            EXAMPLE_MYSQL: ${EXAMPLE_MYSQL:-true}
+            EXAMPLE_DB_USER: ${EXAMPLE_DB_USER:-test}
+            EXAMPLE_DB_PASSWORD: ${EXAMPLE_DB_PASSWORD:-test}
+            EXAMPLE_DB_NAME: ${EXAMPLE_DB_NAME:-test}
+            EXAMPLE_SSL_MODE: verify-full
+            EXAMPLE_TLS_ROOT_CERT: /ssl/root.crt
+            EXAMPLE_TLS_KEY: /ssl/acra-client.key
+            EXAMPLE_TLS_CERT: /ssl/acra-client.crt
+        # We use internal 'python-mysql' network for Django and
+        # DB interconnection, 'python-server' for access to AcraServer
+        # and external network 'world' for port exposing
+        networks:
+            - python-server
+            - python-mysql
+            - python-postgresql
+        volumes:
+            - ../acra/examples/python:/app
+
+    phpmyadmin:
+        build:
+            context: ../
+            dockerfile: ./_common/phpmyadmin/phpmyadmin.dockerfile
+            args:
+                VCS_URL: ${COSSACKLABS_ACRA_VCS_URL}
+                VCS_REF: ${COSSACKLABS_ACRA_VCS_REF}
+                VCS_BRANCH: ${COSSACKLABS_ACRA_VCS_BRANCH}
+                BUILD_DATE: ${COSSACKLABS_ACRAENGDEMO_BUILD_DATE}
+        networks:
+            - webui-mysql
+            - world
+        depends_on:
+            - mysql
+        ports:
+            - 8080:80
+        environment:
+            - PMA_ARBITRARY=1
+            - PMA_HOST=mysql
+            - PMA_USER=test
+            - PMA_PASSWORD=test
+
+    prometheus:
+        image: prom/prometheus:v2.4.0
+        depends_on:
+            - acra-server
+        ports:
+            - "9090:9090"
+        volumes:
+            - ../_common/prometheus:/app.cfg
+        networks:
+            - python-server
+            - world
+        command: >-
+            --config.file=/app.cfg/prometheus.yml
+            --storage.tsdb.path=/prometheus
+            --web.console.libraries=/usr/share/prometheus/console_libraries
+            --web.console.templates=/usr/share/prometheus/consoles
+
+    grafana:
+        image: grafana/grafana:5.2.4
+        depends_on:
+            - prometheus
+        environment:
+            GF_AUTH_ANONYMOUS_ENABLED: !!str 'true'
+            GF_PATHS_PROVISIONING: '/app.cfg/provisioning'
+        ports:
+            - "3000:3000"
+        volumes:
+            - ../_common/grafana:/app.cfg
+        networks:
+            - grafana-prometheus
+            - world
+
+
+    jaeger:
+        image: jaegertracing/all-in-one:1.7
+        environment:
+            SPAN_STORAGE_TYPE: memory
+        ports:
+            - "16686:16686"
+        networks:
+            - python-server
+            - world
+
+    postgresql:
+        # Build and run the container based on the official postgresql image
+        # with an additional configuration in accordance with djangoproject
```

Comment:
lets remove this comment. there is no djangoproject. maybe leave mention that it is official postgresql with some additions (ssl certs?)